### PR TITLE
Create common prerequisites readme

### PR DESCRIPTION
**Pull request by ryansteakley**
**Description of your changes:**
 The prerequisite sections of the cognito and rds-s3 readmes were identical to avoid duplication created a comman readme for this section

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by akartsky — APPROVED** [review]

File: docs/deployment/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   
+1. Created an EKS cluster
+    - If you do not have an existing cluster, run the following command to create an EKS cluster. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
+    - Substitute values for the CLUSTER_NAME and CLUSTER_REGION in the script below
+        ```
+        export CLUSTER_NAME=$CLUSTER_NAME
+        export CLUSTER_REGION=$CLUSTER_REGION
+        eksctl create cluster \
+        --name ${CLUSTER_NAME} \
+        --version 1.19 \
```
Comment (goswamig) [inline docs/deployment/prerequisites.md:24]: can we please update this to 1.20

**Review by goswamig — COMMENTED** [review]
minor change, lets move to k8s 1.20

File: docs/deployment/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      - :warning: Kubeflow is not compatible with the latest versions of of kustomize 4.x. This is due to changes in the order resources are sorted and printed. Please see [kubernetes-sigs/kustomize#3794](https://github.com/kubernetes-sigs/kustomize/issues/3794) and [kubeflow/manifests#1797](https://github.com/kubeflow/manifests/issues/1797). We know this is not ideal and are working with the upstream kustomize team to add support for the latest versions of kustomize as soon as we can.
+   - [python](https://www.python.org/downloads/) - A programming language used for automated installation scripts.
+   - [pip](https://pip.pypa.io/en/stable/installation/) - A package installer for python.
+   
+1. Created an EKS cluster
+    - If you do not have an existing cluster, run the following command to create an EKS cluster. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
+    - Substitute values for the CLUSTER_NAME and CLUSTER_REGION in the script below
```
Comment (surajkota) [inline docs/deployment/prerequisites.md:18]: `CLUSTER_NAME` and `CLUSTER_REGION`

add `--with-oidc` in this command

File: docs/deployment/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - [kustomize version 3.2.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0) - A command line tool to customize Kubernetes objects through a kustomization file.
+      - :warning: Kubeflow is not compatible with the latest versions of of kustomize 4.x. This is due to changes in the order resources are sorted and printed. Please see [kubernetes-sigs/kustomize#3794](https://github.com/kubernetes-sigs/kustomize/issues/3794) and [kubeflow/manifests#1797](https://github.com/kubeflow/manifests/issues/1797). We know this is not ideal and are working with the upstream kustomize team to add support for the latest versions of kustomize as soon as we can.
+   - [python](https://www.python.org/downloads/) - A programming language used for automated installation scripts.
+   - [pip](https://pip.pypa.io/en/stable/installation/) - A package installer for python.
+   
+1. Created an EKS cluster
+    - If you do not have an existing cluster, run the following command to create an EKS cluster. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
```
Comment (surajkota) [inline docs/deployment/prerequisites.md:17]: add another point

- Various controllers in Kubeflow deployment use [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)(IRSA). An OIDC provider must exist for your cluster to use IRSA. 

Add a note
- **Note:** If you are using an existing cluster, Create an OIDC provider and associate it with for your EKS cluster by running the following command:
```
eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
--region ${CLUSTER_REGION} --approve
```

**Pull request merged**